new/components/manager/config/provider_context

```diff
@@ -1 +1 @@
pj8IiLAzsiZhnFBvpUNNSyvU2yXisncG/YxPi\\nrEbFRP3Cksovw4oX01DKhajbo5xgRszZ7252TXnEghWmCYi4MQb0pgxWsWg8V71eD1V91cE8AgRL\\n16ygYAlrqbkauS0T9dvK2DmPJDK1u8PctGinEUks2nWn3nPXfK48uFiuZEZjXJrN+JFHeWxxOJG9\\nJQPB0qxvPhRuSRnVnyd168W+7FzzapKYkzzKqkpZkKe+UcyKhcpuUUHA23S0O6QdEyCHGBRsVn9l\\nV9YTXh7UncvBPGTyoQ9Gm/G8q+fY6uoqTM86IJpkwONZru3cu97nXz7+/cPnXqPevTHCvqsFgTfk\\nj0Le0uI5QPD1sBRgooR4Lni8pZ36WnvncqAmqrWlwfpynz5OqZG+84W6J+9RyPsKEdkt1RVJWW91\\nfak+1RKce4nyEvVtz3S10X5eE1vNm+YF0Qvity6INd/xGQli1RqugglfzHG171c43+O6POa5yj2z\\n7K0PGe3vEeM82Y6zV39nteJu1TH7q8QX78JtUYrr+nhOc6Bl0BkXK3HDIap2ZOUSN4M9NVCdIbZd\\nHlTV0va40ndx2P3A49h+b8q50NWobSgn/QQGeF8/03n98JywyCmPTcyPrnYdSIMqMP0g+m+LSFrE\\nTGAM1a62heB52pBNx/GQmnnFKqB/zlG2pMg0guSS51GIYtAASgsglkmkAyjQnAv94tOSRxTpDr4k\\n8JsKqkBqLw+5vVX4I2SPLMyBUav+F+MuYBKrl7DetgDlMqPQIMVuv6IBq2VuA9S/6jrwkC2AlCxZ\\nIFf9EgVLGjwgZpDR4BiwmYZfdanBcDHn9e7uit/vN3v9ZB4hNRilQl9SRB/VQHXUmgFM5IECtZKM\\nIxgNkoUuSMmoQLPPAkhnT5QmuL5LU8JxPRxu9v/ZoFvgingSrRymEQyn8skTkY46NATOiIhRxTWi\\nQdMwXFkYrgoYkFSTbigRWXDgFjT8FzREqHweQHUwThRiQS4EYG7ZWrd6G1JEn2KiuD1jEfuDhm8u\\nSA14OmTpTIZpngJQeAZmV7hXlzMOQglsAgbnRlTNCzPVu+pKUqBbxgKJuOk7WwK9QJRDw/IfVbs/\\n/oxitVkYuGFumgPYZmYI6pFUq42eWBTBCAHumQcGaRiGXIkFUBdYI6nrReugajfAetOLRECfLV2E\\nHEnulEm7FPcqB0qaMqRJUMqxacUMhmyF3VZnpbEWQfwHSFjEgaCkAlycgwACpjMdXUaVMjDcrxJD\\nY1Alh42sfAd6u8DUQrVBVsPKCkU0O79pvafMDqVN1LCJgJ9YNAdPf7KJj1CCpGDXMwOy8CCZB2pM\\ngFlkZ5U+0qM11qYiMBKqXRoZdj/FKQkyIIf+1Vr4lu76pxhsBgRLKWjNtImWRO7W2HVWv6WuLpWT\\n19JeS3st7bX0QdapPi6QZ4X/wYUkMsM8Q4Ka/8D4WpRqqQaK3j2353XriHbE1/R6dtMrWmoNh+zK\\nQtfZnF6DtxyotI63zqx2XaHxJZQvCcteH4h3R6R1vZHhTz/9NDVdPIJAuoOxMAJZwjRAuOkHw6HU\\nmUJhbmr+jEd4aJRMWb1s9m50iYb31ZC5vR0++54ZdrBaCarIVasnpssPoumhNULhIqxLqIGncqZW\\n/9qaGsjmlTZ5W3UL/c3aSukRAcBnJlSx0hZ+eALwmorhuSI2YfPh3f2GTHW+pQ5oz1Y2ZogmSu2K\\nkIo2Vds8CqTpAuQDPhoYMQdmC1iIN2o7r6wO8tdr+k1Q1rUjQpv8U8+XlEQvIjSBMYNVLSyo47Wx\\nYvLZQpB02UL4kJFFAnM8drqqNpP+h2dqsoLJwBxfaajmv9nKIdiCRIK5CEaFc2uta1K6MWX8tmRu\\n3gHTUddS0+E+dKmmQt2XNJWUv3UC7NNhPR3uvl3W8gy/sNO1vyTFX5LiL0nxycj8JSk+e5a/JMVf\\nkuIvSfGXpPhLUvwlKf6SFH9Jir8kxV+S4i9J2XpJysku5/BXrBzxipX2ev4+FJ+Cwqeg8PeheHHx\\n4uLFxd+H4u9D8feh+PtQ/H0o/j4Ufx+Kt8b8fSj+PhRvsvn7UPx9KP4SA3+Jgb/EwN+H4lWJVyVe\\nlfj7UHy2IH8fir8Pxd+H4u9DOWoOJO8a9K5Bf7XG+Vyt4a8kehdXEtXPstfSydnsEV0Z5ezj4ph1\\nLUfJG+RYcQANmgAVqVbqJ5DL0/GNk8mV09+bR49/czkfdC1dSb0152IjH0hH4oxPJlWGzonhGptR\\nZKEOdV4b9LSE/0jC4S2BCmXxJXHZNCRAXFZnEhUZIhBL1NHpBRBJ2hwajewx7fiooamyplg4TIYS\\nNXtV9UUVvdGwLWvGJ10F2SoGj5g8qDwwgv4vB86pTixr0ccPnz4rgGNDRXPKPchlxuNutmyna1kN\\nAfmQzpEAVGVJneLla18SGSxpTBQTSBDQFJhgCacM0Rdh/2thuR5CAV0ufHHFKYCUrCJOwqP0AWwc\\n3t7eqM/RdDKqfppyO74YjW9up8PJaDwaXwxHk9H05gINT0GAXM3XCF3Yza/O93Y9t6gUn++kPP+A\\nyqmtmWdjM73KD+sfvMx8i/KvWH9VGJODry7/0+mkW/5BXJz8K10wHN1OrqcXaOLl/634X5P6I+v/\\n0fXwtq7/x8Pbm6nX/yfR/4dkBXuZ53FXEjG9IGpxHNqlbjMirZaHrR4QZNMZidpy6lRegiIgqSUU\\nqXjQT2n85v4Ea3abeFjp8mCafHy59uc7Grckz3ePtgRcVbcJdOMqlqkzsVuZVW26T3a41qRvfh4/\\nkv4vaX9E+/96OKrM//DeeDwcT7z+fyP9f7/lBG+7Tt6UeLMDuCWD5aukvjp5eBH2xRdffPHFF198\\n8cUXX3zxxRdffNla/g/Hv4N/AMgAAA==\\n"}
```

Comment:
this file should be deleted. we set the provider context within the blueprint or from the cli (depending on the key).